caddy/caddy.go

```diff
@@ -25,7 +25,7 @@ const defaultDocumentRoot = "public"
 
 func init() {
 	caddy.RegisterModule(FrankenPHPApp{})
-	caddy.RegisterModule(FrankenPHPModule{})
+	caddy.RegisterModule(FrankenPHPModule{ResolveRootSymlink: true})
```

Comment:
I don't think this is right. Shouldn't pass options here.

The typical way to have a boolean that you can detect not having been specified by the user is by using a `*bool` instead so the default is actually nil.

Then you can set the value in Provision based on whether it was user-specified or not.

Reply:
Fixed, thanks!